Formula/istioctl.rb

```diff
@@ -17,31 +17,26 @@ class Istioctl < Formula
   depends_on "go" => :build
   depends_on "go-bindata" => :build
 
+  # Build script bin/init.sh runs curl command with `--retry-connrefused`,
+  # which needs curl >= 7.52. Can remove if Linux CI default curl is updated.
+  uses_from_macos "curl" => :build
```

Comment:
This seems off. We should probably check that whatever it's downloading is versioned and checksummed. If not, that needs to be a `resource`.

Reply:
Looks like `envoy` and `wasm` plugins: https://github.com/istio/istio/blob/1.10.3/bin/init.sh#L129-L145

For `envoy`:
- macOS uses v1.0.2 https://github.com/istio/istio/blob/1.10.3/Makefile.core.mk#L155
- Linux uses some SHA https://github.com/istio/istio/blob/1.10.3/istio.deps

Haven't taken much of a look yet for dealing with this.